website/docs/r/record.html.markdown

```diff
@@ -92,7 +92,12 @@ resource "ns1_record" "www" {
 The following arguments are supported:
 
 * `zone` - (Required) The zone the record belongs to.
+Cannot have leading or trailing dots (".") and can be cleaned with:
+```
+zone = replace(".terraform-test-zone.io.", "/(^\\.)|(\\.$)/", "")
+```
```

Comment:
couple suggestions here. Working an example of replace in the main example above would be cool, I think. Something that shows doing it to a variable and in context of a full config. Something like what i have here, I was thinking https://github.com/ns1/ns1-go/pull/80 - i wanna make sure it's foolproof, obvious, and easy.

Once it's in the main example you could just say "see the example above" or whatever. If it feels like more than that could be useful, I'd put further details in it's own little section below like we do for filters and meta in this file. Some language like "systems can have different requirements for FQDN, a common thing is to return or require a trailing dot, e.g. `foo.com.`, other systems handle that for you. NS1 doesn't not require or return a trailing dot. So, depending on what resources you are connecting, a little bit of replacement might be needed <examples ensue>"